streaming/sender.c

```diff
@@ -214,7 +214,21 @@ static int rrdpush_sender_thread_connect_to_parent(RRDHOST *host, int default_po
 
     char http[HTTP_HEADER_SIZE + 1];
     int eol = snprintfz(http, HTTP_HEADER_SIZE,
-            "STREAM key=%s&hostname=%s&registry_hostname=%s&machine_guid=%s&update_every=%d&os=%s&timezone=%s&abbrev_timezone=%s&utc_offset=%d&hops=%d&tags=%s&ver=%u"
```

Comment:
@vkalintiris we are modifying the stream handshake, probably it is necessary to also change streaming version here.

Reply:
I'm not sure if we need to update the streaming version because we are only adding a new key/value pair, ie. we are not modifying the protocol itself (at least the hop count PR didn't bump the protocol's version https://github.com/netdata/netdata/pull/11311)